docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -762,11 +762,15 @@ disable this behaviour using `CanonicalURLMiddleware::singleton()->setForceBasic
 configuration in YAML.
 
 We also want to ensure cookies are not shared between secure and non-secure sessions, so we must tell Silverstripe CMS to 
-use a [secure session](https://docs.silverstripe.org/en/3/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie). 
+use a [secure session](https://docs.silverstripe.org/en/3/developer_guides/cookies_and_sessions/sessions/#secure-session-cookie).
+It is also a good idea to set the `samesite` attribute for the session cookie to `Strict` unless you have a specific use case for
+sharing the session cookie across domains.
+
 To do this, you may set the `cookie_secure` parameter to `true` in your `config.yml` for `Session`
 
 ```yml
 SilverStripe\Control\Session:
+  cookies_samesite: 'Strict'
```

Comment:
```suggestion
  cookie_samesite: 'Strict'
```
Remove the plural to match `cookie_secure`.  You'll need to update this in a bunch of places